website/static/js/projectCreator.js

```diff
@@ -20,16 +21,31 @@ var CREATE_URL = '/api/v1/project/new/';
     * Params:
     *  - data: Data to populate the template selection input
     */
-function ProjectCreatorViewModel(params) {
+function ProjectCreatorViewModel(params) {   
     var self = this;
     self.params = params || {};
     self.minSearchLength = 2;
     self.title = ko.observable('');
     self.description = ko.observable();
+
+    self.category = ko.observable('project');
+    self.NOCATEGORY = '------';
+    self.categoryMap = nodeCategories;
+    self.categories = [self.NOCATEGORY].concat(Object.keys(nodeCategories));
+
     self.errorMessage = ko.observable('');
 
     self.hasFocus = params.hasFocus;
 
+    self.usingTemplate = ko.observable(false);
+
+    $('#createNodeTemplates').on('change', function() {
```

Comment:
I'm not sure this is necessary. Why reset the category input if the user selects a template?